secrets-store-csi-driver-provider-gcp.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: secrets-store-csi-driver-provider-gcp
+  # When bumping the version check if the GHSA mitigations below can be removed.
+  version: 1.2.0
+  epoch: 0
+  description: Google Secret Manager provider for the Secret Store CSI Driver.
+  copyright:
+    - license: Apache-2.0
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - busybox
+      - build-base
+      - go
+      - ca-certificates-bundle
+pipeline:
+  # We can't use go/install because this requires a specific go version
```

Comment:
It does? Which version?

Reply:
Whoops, fixed the comment. It was about specific ldflags to set the right version.